kura/distrib/src/main/resources/Win32-nn/Kura.nsi

```diff
@@ -0,0 +1,435 @@
+!include "Library.nsh"
```

Comment:
Yes, the ReplaceInFile function was copied from that public library.
The rest is using standard NSIS functions and libraries, so you might consider a whole NSIS CQ if needed.